pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
Which indeed I still haven't understood. 
I can see the point in letting user select between adaptive and normal, albeit (50/60hz mumblings and [quite](https://www.hardocp.com/article/2012/04/16/nvidia_adaptive_vsync_technology_review/3) [random](http://techreport.com/review/22735/a-closer-look-at-some-geforce-gtx-680-features/2) [confusion](https://hardforum.com/threads/nvidia-adaptive-vsync-vs-vsync-triple-buffering.1688239/#post-1038642454) [aside](https://forums.geforce.com/default/topic/517715/do-you-need-tripple-buffering-with-the-new-adaptive-vsync-option-enabled-/#3683196)) I still feel like one of them has to be 100% better than the other ultimately. 

But wtf is compositor vsync? It's not like being subject to dwm is in program's control.